### PR TITLE
Update global param sample

### PR DESCRIPTION
**Pull request by tbohn**
Addresses issue #336.

### CODE REVIEW

**Comment by jhamman** [comment]
looks good @tbohn - 

This merge is blocked by #341 but will not require a rebase.  

Can you also update this page in the docs: http://vic.readthedocs.org/en/develop/Documentation/Drivers/Classic/GlobalParam/#example-global-parameter-file

I think the table is mostly okay but the example is also out of date.

**Comment by tbohn** [comment]
OK, will do.

**Comment by tbohn** [comment]
I've added my edits to GlobalParam.md to this PR.  I've also merged in all the other changes to develop that have been happening.  Perhaps I shouldn't have done that...

File: docs/Documentation/Drivers/Classic/GlobalParam.md [hunk 1]
```diff
@@ -286,12 +289,12 @@ The following options are no longer supported.
 #######################################################################
 NLAYER      3   # number of soil layers
 NODES       10  # number of soil thermal nodes
-MODEL_STEPS_PER_DAY  8   # model time step in hours (set to 1 if FULL_ENERGY = FALSE, set to > 4 if FULL_ENERGY = TRUE)
-SNOW_STEPS_PER_DAY  8   # time step in hours for which to solve the snow model (should = MODEL_STEPS_PER_DAY if MODEL_STEPS_PER_DAY > 1)
+MODEL_STEPS_PER_DAY  8   # number of model time steps per day (set to 1 if FULL_ENERGY = FALSE, set to > 4 if FULL_ENERGY = TRUE)
```
Comment (jhamman) [inline docs/Documentation/Drivers/Classic/GlobalParam.md]: change 4.2 to 5.0 in header line above.

**Comment by jhamman** [comment]
you're fine keeping this branch in sync with develop.  Because you created this branch off of your wb fix branch, we'll need to merge that first but this is fine.

File: docs/Documentation/Drivers/Classic/GlobalParam.md (hunk 1 above)
Comment (tbohn) [inline docs/Documentation/Drivers/Classic/GlobalParam.md]: Done.

**Pull request merged**